core/src/main/scala/Config.scala

```diff
@@ -559,12 +563,8 @@ object Config {
         def checkServerTrusted(certs: Array[X509Certificate], authType: String): Unit = ()
       }
 
-    def getKubernetesPodCert(): Option[SSLContext] = {
-      // Auto-mounted at this path for pods
-      // https://kubernetes.io/docs/tasks/access-application-cluster/access-cluster/#accessing-the-api-from-a-pod
-      val certBundle = "/var/run/secrets/kubernetes.io/serviceaccount/ca.crt"
-
-      val is = new FileInputStream(certBundle)
+    def getKubernetesPodCert(certPath: Path): Option[SSLContext] = {
```

Comment:
Just so I understand what's going on here, you're assuming that there's per-cluster PKI for interacting with the Kube API? Presumably this means the location that Nelson is deployed needs the certs on disk (this should be fine, just trying to grok it)

Reply:
Yep that is the current assumption